build.py

```diff
@@ -37,12 +35,16 @@ def updateZip(zipname, filename, filedata):
     with zipfile.ZipFile(zipname, mode='a', compression=zipfile.ZIP_DEFLATED) as zf:
         zf.write(filedata, filename)
 
-
 if not os.path.exists(ORIGINAL_FILE_NAME):
-    with urllib.request.urlopen('https://jeff.tdrealms.com/Add-Ons/Eloquence.nvda-addon') as response:
-        with open(ORIGINAL_FILE_NAME, "wb") as f:
-            shutil.copyfileobj(response, f)
-shutil.copyfile(ORIGINAL_FILE_NAME, FILE_NAME)
+    print("Cannot find original Eloquence file. Will retrieve it manually.")
+    buildSystemPathToEloquence = r"%s\nvda\addons\Eloquence" %(os.getenv('APPDATA'))
```

Comment:
Not sure if there is a better way to find appdata or work with paths in Windows.